models/spring-ai-bedrock/src/main/java/org/springframework/ai/bedrock/llama/api/LlamaChatBedrockApi.java

```diff
@@ -226,7 +226,42 @@ public enum LlamaChatModel implements ChatModelDescription {
 		/**
 		 * meta.llama3-70b-instruct-v1:0
 		 */
-		LLAMA3_70B_INSTRUCT_V1("meta.llama3-70b-instruct-v1:0");
+		LLAMA3_70B_INSTRUCT_V1("meta.llama3-70b-instruct-v1:0"),
+
+		/**
+		 * meta.llama3-1-8b-instruct-v1:0
+		 */
+		LLAMA3_1_8B_INSTRUCT_V1("meta.llama3-1-8b-instruct-v1:0"),
+
+		/**
+		 * meta.llama3-1-70b-instruct-v1:0
+		 */
+		LLAMA3_1_70B_INSTRUCT_V1("meta.llama3-1-70b-instruct-v1:0"),
+
+		/**
+		 * meta.llama3-1-405b-instruct-v1:0
+		 */
+		LLAMA3_1_405B_INSTRUCT_V1("meta.llama3-1-405b-instruct-v1:0"),
+
+		/**
+		 * meta.llama3-2-1b-instruct-v1:0
+		 */
+		LLAMA3_2_1B_INSTRUCT_V1("meta.llama3-2-1b-instruct-v1:0"),
```

Comment:
it's hard to find these strings from the docs.  I went [here](https://github.com/meta-llama/llama-stack/blob/8d41e6caa90c0e7bb44ec1755302a943bdf654f1/llama_stack/providers/adapters/inference/bedrock/bedrock.py#L23) to find the previously listed `llama3-1-x` models but dont' see the `llama-2-x` model values in the same code base.

Reply:
There is also cross-region ids, see - https://docs.aws.amazon.com/bedrock/latest/userguide/cross-region-inference-support.html

Reply:
ah, found the model listing here - https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html  added to the javadoc for future reference.